src/BenchmarksApps/Kestrel/PlatformBenchmarks/Data/RawDb.cs

```diff
@@ -203,7 +205,32 @@ public async Task<World[]> LoadMultipleUpdatesRows(int count)
                     await reader.NextResultAsync();
                 }
             }
+
+            using (var connection = await _updateDataSource.OpenConnectionAsync())
```

Comment:
Curious: we don't use `DisposeAscyn`?

Reply:
Nah :) For cases where we know we don't need to do I/O, we somtimes just use the sync APIs, since they may be (very) slightly more efficient. Dispose only returns the connection to the pool, so no need for async here (for this benchmark).

Reply:
I could troll you on this. But that's not in my nature. ;-)

Reply:
Is that so :)

In any case,it's been a very long time since I've checked if there's any meaningful difference between the async and sync versions - it may be there's nothing there any more... Even if there is, it's very unlikely to be interesting to anyone except for very high-perf scenarios like these benchmarks.